shared/src/packages/projects/libs/enums/projects-api-path.enum.ts

```diff
@@ -1,5 +1,7 @@
 const ProjectsApiPath = {
   ROOT: '/',
+  PROJECT_$PROJECT_ID: '/project/:projectId',
+  AVATAR: '/avatar',
```

Comment:
But you're using them as a joined single path?

Reply:
![image](https://user-images.githubusercontent.com/87016333/229593197-73bc90fd-72bb-49ff-85a3-ef46afd6c699.png)
like that

Reply:
idk, just in case to reuse it. Gonna fix it. Also maybe remove first project in path as it defined in controller root projects. so it is going to be projects/project/:projectId/avatar ?

Reply:
```suggestion
   $PROJECT_ID_AVATAR: '/:projectId/avatar',
  ```

Reply:
let's better have /project/:projectId/avatar, k